### PR TITLE
DVCSMP-1795 Philips Hue: Add support for Hue Ambience bulb

### PR DESCRIPTION
**Pull request by larsfinander**
-Add support for Hue White Ambiance Bulb (white,dimmable+color temp)

### CODE REVIEW

**Comment by larsfinander** [comment]
@juano2310 please review! New Hue light type (white + temperature control)

File: devicetypes/smartthings/hue-white-ambiance-bulb/hue-white-ambiance-bulb.groovy [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+// for the UI
+metadata {
+    // Automatically generated. Make future change here.
+    definition (name: "Hue White Ambiance Bulb", namespace: "smartthings", author: "SmartThings") {
+        capability "Switch Level"
+        capability "Actuator"
+        capability "Color Temperature"
+        capability "Switch"
+        capability "Refresh"
+        capability "Sensor"
```
Comment (juano2310) [inline devicetypes/smartthings/hue-white-ambiance-bulb/hue-white-ambiance-bulb.groovy]: I'm sure we have that in all of them but should we keep sensor here?

**Comment by juano2310** [comment]
I think all the rest looks fine 👍

File: devicetypes/smartthings/hue-white-ambiance-bulb/hue-white-ambiance-bulb.groovy (hunk 1 above)
Comment (larsfinander) [inline devicetypes/smartthings/hue-white-ambiance-bulb/hue-white-ambiance-bulb.groovy]: Fixed

**Comment by juano2310** [comment]
👍

**Pull request merged**